bodo/runtests_caching.py

```diff
@@ -35,6 +121,33 @@
 if "NUMBA_CACHE_DIR" in os.environ:
     del os.environ["NUMBA_CACHE_DIR"]
 
+ic_queue = multiprocessing.Queue()
```

Comment:
Adding a test to a runner file is confusing. Can this be moved to a regular pytest file?